op-service/tri/tri.go

```diff
@@ -0,0 +1,73 @@
+// Kleene’s three-valued logic (K3) enlarges classical logic with an “undefined”
+// value ⊥ alongside true (T) and false (F) to model partial or indeterminate
+// statements. It keeps the classical truth tables where both operands are T or
+// F, but whenever any input is ⊥, the result is ⊥ except where truth can still
+// be decided. Only T is “designated” (counted as logically valid), so the law
+// of excluded middle (p ∨ ¬p) may be ⊥ rather than T, reflecting uncertainty,
+// while contradictions (p ∧ ¬p) can also be ⊥ instead of always F. This makes
+// K3 paracomplete – avoiding commitment to truth or falsity when information is
+// incomplete—yet it reduces to classical logic once all statements are decided.
+
```

Comment:
```suggestion
//
```

Otherwise the top part will be considered a license statement by the go tool, and only the part below this blank line will render as package go doc comment.